movie-app/functions/fetch-search-movie/fetch-search-movie.js

```diff
@@ -0,0 +1,21 @@
+const axios = require('axios');
+
+const handler = async (event) => {
+  const { searchTerm } = event.queryStringParameters;
+  const API_KEY = `api_key=${process.env.API_KEY}`;
+  const BASE_URL = 'https://api.themoviedb.org/3';
+  const url = `${BASE_URL}/search/multi?${API_KEY}&include_adult=false&query=${searchTerm}`;
```

Comment:
Search result should not include tv shows and actor/actress since we don't have infrastructure for them yet

Reply:
I fixed it! Thank you.